src/xkernel.cpp

```diff
@@ -135,8 +137,10 @@ namespace xeus
         p_server = sbuilder(*p_context, m_config, m_error_handler);
         p_server->update_config(m_config);
 
+        std::clog << "Before instantiating debugger" << std::endl;
```

Comment:
Do we need to keep these logs? Or were they meant for debugging?

Just noticed they are showing in `jupyterlite-xeus`: https://github.com/jupyterlite/xeus/issues/101

Reply:
These are leftover, we should remove them